org/ros/rosjava_messages/octomap_msgs/0.3.2/octomap_msgs-0.3.2.pom

```diff
@@ -9,7 +9,7 @@
     <dependency>
       <groupId>org.ros.rosjava_bootstrap</groupId>
       <artifactId>message_generation</artifactId>
-      <version>[0.1,0.2)</version>
+      <version>[0.1, )</version>
```

Comment:
Hmm as of Kinetic branch, this should say [0.3, 0.4) if I'm not mistaken.

Could it be a problem when generating this artifact?

Reply:
That's right! you already told me if I'm not mistaken. I'll submit the updated pull request tomorrow.